frontend/test/test-minimal-modules.h

```diff
@@ -98,6 +98,11 @@ module ChapelArray {
 
   proc chpl__buildIndexType(d: domain) type do
     return chpl__buildIndexType(d.rank, d.idxType);
+
+  // Module code needed to avoid fatal errors in resolution
+  param nullPid = -1;
+  proc _isPrivatized(value) param do return false;
+  record dmap { }
```

Comment:
Hm, I just tested and there's no fatal error in resolution from leaving out any single one of these. It does cause `testArrays` to fail from resolution errors, since these symbols live in the real `ChapelArray` and are needed. I think either I wrote this comment and called that a "fatal error" incorrectly, or maybe there truly was a fatal error some N resolution bugfixes ago that have since gone in. Either way, I'll just remove the comment.